skill-tree /src/test/java/com/RDS/skilltree/EndorsementsIntegrationTests.java

```diff
@@ -9,12 +11,15 @@
 import org.junit.jupiter.api.DisplayName;
 import org.springframework.beans.factory.annotation.Autowired;
 
+import java.util.UUID;
+
 import static io.restassured.RestAssured.given;
 import static org.hamcrest.Matchers.*;
 
 public class EndorsementsIntegrationTests {
     @Autowired
-    private ObjectMapper objectMapper;
+    private ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
`@Autowired` annotation looks out of place. There is also a trailing additional `;`